docker_tests/test_docker_compose_quick_start.py

```diff
@@ -103,6 +104,17 @@ def wait_for_terminal_dag_state(dag_id, dag_run_id):
             break
 
 
+ALL_CONTAINERS = [
```

Comment:
Is there any way we can fetch this list programmatically instead of hard-coding the names here?

Reply:
sure - why not :)